docs/self-hosting/advanced/auth/clerk.mdx

```diff
@@ -27,6 +27,7 @@ Go to [Clerk](https://clerk.com?utm_source=lobehub\&utm_medium=docs) to register
   ```shell
   NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=pk_live_xxxxxxxxxxx
   CLERK_SECRET_KEY=sk_live_xxxxxxxxxxxxxxxxxxxxxx
+  NEXT_PUBLIC_ENABLE_NEXT_AUTH=0
```

Comment:
I think a better way is to add a Callout to show the self-built docker image need this env? as most clerk user doesn't need to add this env

Reply:
In that case, it would be nice to clarify, when the env is needed and when it can be ignored.
The release note i provided says:

> it's necessary to add NEXT_PUBLIC_ENABLE_NEXT_AUTH if self hosting with docker.